### ISSUE TITLE
Add drag/drop for receipt empty state

### ISSUE THREAD

**Issue by JmillsExpensify** [Expensify/App#27744]
If a `Collect` or `Control` policy is configured with any receipt-related category violation, we’ll update the confirmation screen for manual requests and include a receipt empty state. Tapping the empty state brings up the camera (mobile) or file picker (web). For web, we’ll support drag/drop.

![image](https://github.com/Expensify/App/assets/1127863/be806bc8-ac27-4ab2-adbb-acac07c6fc3a)

<details><summary>Upwork Automation - Do Not Edit</summary>
    <ul>
        <li>Upwork Job URL: https://www.upwork.com/jobs/~01e09455d07e462ca6</li>
        <li>Upwork Job ID: 1704005352336375808</li>
        <li>Last Price Increase: 2023-09-19</li>
    </ul>
</details>

**Comment by cead22** [comment]
No update from me, focused on detailed design for Wave 9: Violations

**Comment by JmillsExpensify** [comment]
Still working through the detailed design for Wave 9.

**Comment by JmillsExpensify** [comment]
Still working through the detailed.

**Comment by JmillsExpensify** [comment]
Still focused on violations implementation first.

**Comment by cead22** [comment]
Still on HOLD, but Infinite Red started on App#30663

**Comment by cead22** [comment]
App#30663 is merged so this is off hold. @cdanwards @trevor-coleman can you comment here so I can assign you?

**Comment by cdanwards** [comment]
Commenting for assignment

**Comment by JmillsExpensify** [comment]
Waiting on PR

**Comment by cead22** [comment]
@cdanwards any updates on this?

**Comment by cead22** [comment]
@cdanwards is working on a couple of other items for violations. This is in the queue, but not being worked on yet

**Comment by JmillsExpensify** [comment]
All good, I think it's lower priority.

**Comment by cead22** [comment]
This is in the queue, but not being worked on yet

**Comment by JmillsExpensify** [comment]
Still not at the top of the priority list.

**Comment by cdanwards** [comment]
Beginning work on this.

**Comment by shawnborton** [comment]
One small design bug to fix while we're here:
- let's update the border color of the thumbnail area to use our `colors.border` value
- let's update the background color of the thumbnail area to use our `colors.highlightBG` value

**Comment by cead22** [comment]
@cdanwards please tag @Expensify/design in the PR when you create it

**Comment by shawnborton** [comment]
Also, the border should not be 2px like it is here:
![CleanShot 2024-01-25 at 14 42 36@2x](https://github.com/Expensify/App/assets/2319350/11408c09-568b-4782-95d1-9537896b3d20)


Just 1px stroke for the border.

**Comment by shawnborton** [comment]
Also it looks like the image is cut off - I wonder if we need to get you a better image for that? Can you share the image that you are currently using please? Thanks!

**Comment by cdanwards** [comment]
@shawnborton sorry this got lost in my GH notifications. This one has already been merged, but [here](https://github.com/Expensify/App/blob/main/assets/images/empty-state__attach-receipt.svg) is the `svg` that we're using for this component

**Comment by shawnborton** [comment]
Interesting, the svg itself seems fine as we can see the borders:
![CleanShot 2024-02-05 at 11 55 19@2x](https://github.com/Expensify/App/assets/2319350/678679d5-ca98-4af7-a03f-b427770d0659)

Any idea why it might be getting cut off?

**Comment by shawnborton** [comment]
Exactly, see how the 1px stroke is slightly cut off?

**Comment by cdanwards** [comment]
@shawnborton I found the issue. Going to put together a little PR with a fix for that and also the other corrections you mentioned.

**Comment by shawnborton** [comment]
I think something is happening here, there is actually no BG color being shown at all. If there was, we wouldn't see the illustration come through.

**Comment by cdanwards** [comment]
![Screenshot 2024-02-05 at 4 26 10 PM](https://github.com/Expensify/App/assets/8878532/8c3743bd-0928-4ccc-ae51-e6b87ff514b0)
There we are. Got the right values!

**Comment by shawnborton** [comment]
Much better!

**Comment by JmillsExpensify** [comment]
I believe all the PRs on this issue are merged right? Can we close now?

**Comment by cead22** [comment]
Yes! @cdanwards reopen if I missed anything
